utils/build/docker/weblog-cmd.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+
+##########################################################################################
+# This is an entrypoint file that all containers run in system-tests are funneled through.
+##########################################################################################
+
+set -eu
+
+# Zero is good!
+script_status=0
+
+echo "Configuration script executed from: ${PWD}"
+BASEDIR=$(dirname $0)
+echo "Configuration script location: ${BASEDIR}"
+
+if ! grep -q "#!/bin/bash" "./app.sh"; then
+    echo "Please ensure you add #!/bin/bash to your ./app.sh file"
+    exit 1
+fi
+
+if [ ${SYSTEMTESTS_SCENARIO:-DEFAULT} = "UDS" ]; then
+
+    export EXPECTED_APM_SOCKET=${DD_APM_RECEIVER_SOCKET:-/var/run/datadog/apm.socket}
+    echo "Setting up UDS with ${EXPECTED_APM_SOCKET}."
+    
+    export SOCKET_DIR=$(echo ${EXPECTED_APM_SOCKET} | sed 's|\(.*\)/.*|\1|')
+    mkdir -p ${SOCKET_DIR}
+    chmod -R a+rwX ${SOCKET_DIR}
+
+    ( socat -d -d UNIX-LISTEN:${EXPECTED_APM_SOCKET},fork TCP:library_proxy:${HIDDEN_APM_PORT_OVERRIDE:-7126} > /var/log/system-tests/uds-socat.log 2>&1 ) &
+
+    attempts=$((14))
+    while [ ! grep -q "listening on" "/var/log/system-tests/uds-socat.log" ]
+    do
+        sleep 0.5
+        attempts=$((attempts-1))
+        if [ $attempts -lt 1 ]; then
+            echo "Unable to verify creation of ${EXPECTED_APM_SOCKET}."
+            script_status=1
```

Comment:
To simplify a little bit, you can probably just do an `exit 1` here and remove `script_status` var ? 

Just a fine polishing suggestion though :)